flare/agent_zoo/simple_rl_agents.py

```diff
@@ -69,3 +64,35 @@ def _cts_predict(self, observations, states):
         actions, next_states = self.predict(
             'RL', inputs=dict(sensor=observations), states=dict(state=states))
         return [actions.values()[0][0]], [next_states.values()[0][0]]
+
+
+class ActionNoiseAgent(Agent):
+    """
+    This class extends `SimpleRLAgent` by applying action noise after 
+    prediction. It can be used to algorithms with deterministic policies, e.g.,
+    `DDPG`.
+    """
+
+    def __init__(self, env, num_games, action_noise):
+        super(ActionNoiseAgent, self).__init__(env, num_games)
+        self.action_noise = action_noise
+
+    def _cts_store_data(self, observations, actions, states, rewards):
+        assert len(observations) == 1 and len(actions) == 1
+        self._store_data(
+            'RL',
+            dict(
+                sensor=observations[0], action=actions[0], reward=rewards))
```

Comment:
We could also break line 85 into 3 lines as in simple_rl_agents.py line22-24. This helps to highlight the diff if we add some changes to one type of data, e.g., adding scaling factor to rewards.

Reply:
the automatic style checks did this.

Reply:
You don't have to override this function. Please git pull the most up-to-date code.